prow/external-plugins/cherrypicker/server_test.go

```diff
@@ -570,13 +580,13 @@ func testCherryPickPRWithLabels(clients localgit.Clients, t *testing.T) {
 			},
 			prLabels: []github.Label{
 				{
-					Name: "cherrypick/release-1.5",
+					Name: labelPrefix + "release-1.5",
```

Comment:
nit: Prefer not to parameterize your test cases. This test is not only testing the output of the plugin but also testing that you correctly create test data here at runtime. Passing static data would be preferred.

Reply:
Fixed.